<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageLookupTablesViewControls</class>
 <widget class="QWidget" name="ImageLookupTablesViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <layout class="QGridLayout" name="m_ComboBoxSelectorGridLayout">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="m_LookupTableLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Lookup Table</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_LookupTableComboBox"/>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btn_Save">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/ImageLookupTables.qrc">
           <normaloff>:/ImageLookupTables/btnSave.png</normaloff>:/ImageLookupTables/btnSave.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_New">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/ImageLookupTables.qrc">
           <normaloff>:/ImageLookupTables/btnNew.png</normaloff>:/ImageLookupTables/btnNew.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btn_Load">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/ImageLookupTables.qrc">
           <normaloff>:/ImageLookupTables/btnLoad.png</normaloff>:/ImageLookupTables/btnLoad.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>19</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ctkCollapsibleGroupBox" name="m_RangeGroupBox">
       <property name="title">
        <string>Intensity Range</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QGridLayout" name="m_RangeGridLayout">
          <property name="margin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="m_MinIntensityRangeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>min</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ctkSliderWidget" name="m_MinSlider" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_MaxIntensityRangeLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>max</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ctkSliderWidget" name="m_MaxSlider" native="true"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_LevelLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>level</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ctkSliderWidget" name="m_LevelSlider" native="true"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="m_WindowLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>window</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ctkSliderWidget" name="m_WindowSlider" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="ctkCollapsibleGroupBox" name="m_LimitsGroupBox">
       <property name="title">
        <string>Data Limits</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QGridLayout" name="m_LimitsGridLayout">
          <property name="margin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="m_MinLimitLabel">
            <property name="text">
             <string>min</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="m_MinLimitDoubleSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>the minimum intensity value to use</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="m_MaxLimitLabel">
            <property name="text">
             <string>max</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="m_MaxLimitDoubleSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>the maximum intensity value to use</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="m_LimitsHorizontalSpacerRight">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="m_ResetButton">
            <property name="toolTip">
             <string>reset to the full image data range</string>
            </property>
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_EditLabelsGroupBox">
       <property name="title">
        <string>Edit Labels</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="widget_LabelTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>27</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Color</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="btn_AddLabel">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resources/ImageLookupTables.qrc">
              <normaloff>:/ImageLookupTables/btnAdd.png</normaloff>:/ImageLookupTables/btnAdd.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_RemoveLabel">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resources/ImageLookupTables.qrc">
              <normaloff>:/ImageLookupTables/btnRemove.png</normaloff>:/ImageLookupTables/btnRemove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_MoveLabelUp">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resources/ImageLookupTables.qrc">
              <normaloff>:/ImageLookupTables/btnMoveUp.png</normaloff>:/ImageLookupTables/btnMoveUp.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_MoveLabelDown">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resources/ImageLookupTables.qrc">
              <normaloff>:/ImageLookupTables/btnMoveDown.png</normaloff>:/ImageLookupTables/btnMoveDown.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="m_BottomVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/ImageLookupTables.qrc"/>
 </resources>
 <connections/>
</ui>
